style.css

```diff
@@ -15,11 +15,12 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
 */
 
 /*
- * Control the hover stylings of outline block style.
+ * Control the hover stylings of Button block's Outline style.
  * Unnecessary once block styles are configurable via theme.json
  * https://github.com/WordPress/gutenberg/issues/42794
  */
-.wp-block-button.is-style-outline>.wp-block-button__link:not(.has-background):hover {
+
```

Comment:
I added a space below some of the comments for consistency. It possibly could be removed from all of them instead.